### ISSUE TITLE
ModAdNetworks crashes on launch when user ports it.

### ISSUE THREAD

**Issue by khanbhai89** [apportable/spritebuilder-ModAdNetworks#1]
1. Clone this Project.
2. And open project in Xcode.
3. Port this project to an Android device.
4. Launch Application on the device.
   
   -- Result: Application crashes to android launcher with Unfortunately Stopped pop-up.
     This happened 3/3 times.

**Mac OS X:** 10.9.5. 
**Xcode:** 6.1. 
**SpriteBuilder:** 1.4.9. 
**AndroidPlugin:** SBAndroid Indie 948d0c4e
**Device:** SGS4/4.4.2.

Log: https://drive.google.com/a/werplay.com/file/d/0B5KeRaV3CWUaRGlXQ05JV3FCSU0/view?usp=sharing

**Comment by ebrodzki** [comment]
Was the necessary changes made to the project? The manifest now needs         android:name="com.apportable.BasicRuntimeApplication"
and I believe an additional linker flag is needed as well. If those are not there yet then this is expected

**Comment by khanbhai89** [comment]
@ebrodzki Yes changes were made and it was still occurring with all the apps ADMob & MoPub as well. Retested it on: 
SBAndroidPluginIndie-sbandroid-indie-578c109f92ba6615.1 
Mac OS X: 10.9.5. 
Xcode: 6.1.
SpriteBuilder 1.4.9. (Mac OS X)

**Comment by Shahidammer** [comment]
This issue is re-occurring on BannerCodeSample but it works fine in BannerSample

Mac OS X: 10.10.0
XCode: 6.1.
SBAndroid Pro Version: SBAndroidPluginIndie-sbandroid-indie-578c109f92ba6615.1 Pro
Spritebuilder Version: 1.4.9.

**Comment by khanbhai89** [comment]
Disregard comment above.

This issue is re-occurring.

Mac OS X: 10.10.0
XCode: 6.1.
SBAndroid Version: SBAndroidPluginProfessional-sbandroid-professional-092e93d904850685
Spritebuilder Version: 1.4.9.

**Comment by Shahidwerplay** [comment]
This issue is re-occurring.

Mac OS X: 10.10.0
XCode: 6.1.
SBAndroid Version: SBAndroidPluginProfessional-sbandroid-professional-092e93d904850685
Spritebuilder Version: 1.4.9.

**Comment by delackner** [comment]
If you have _ever_ installed the app using a previous build, you need to delete it from the device before trying again.  The updated manifest with the new field will not properly be copied otherwise.  Perhaps?  I don't have access permission to the log file that you uploaded

**Comment by khanbhai89** [comment]
@delackner Yes we deleted the previous app from the device and tested it on multiple devices as well. Let me know your email I would add you as a viewer on it.

Thanks!

**Comment by delackner** [comment]
Can you double check you have the latest code by pulling? The most recent release requires that the AndroidManifest.xml has, inside <application
        android:name="com.apportable.BasicRuntimeApplication"

Only the MoPub example was actually up-to-date for that change.
